com.unity.shadergraph/Editor/Generation/Processors/GenerationUtils.cs

```diff
@@ -627,6 +627,10 @@ internal static void GeneratePropertiesBlock(ShaderStringBuilder sb, PropertyCol
                     prop.AppendPropertyBlockStrings(sb);
                 }
 
+                sb.AppendLine(Texture2DArrayShaderProperty.kLightmapsArray.GetPropertyBlockString());
+                sb.AppendLine(Texture2DArrayShaderProperty.kLightmapsIndirectionArray.GetPropertyBlockString());
+                sb.AppendLine(Texture2DArrayShaderProperty.kShadowMasksArray.GetPropertyBlockString());
```

Comment:
These shouldn't be in the core generator; that forces them into every shader ever produced by ShaderGraph.  They should be added by the Targets.

Reply:
moved to targets for urp and hdrp.